src/main/java/frc/robot/commands/AlignByAprilTagGyro.java

```diff
@@ -53,6 +55,7 @@ public AlignByAprilTagGyro(DriveSubsystem swerveDrive, double targetX, double ta
     // Called when the command is initially scheduled.
     @Override
     public void initialize() {
+        // Probably a good idea to just make a robot centric drive function that you can call so you don't have to worry about restoring the field-centric state
```

Comment:
Might be good to do that, although I think it means that we need to declare field centric in the robot container instead if we want to be able to toggle between either mode while driving which I dont love. Also us driving field centric is contingent on the gyro having the correct rot which would be hard to check if we wanted to have two distinct functions called elsewhere